### PR TITLE
Update create-and-link-connections.md

### PR DESCRIPTION
**Pull request by RNHTTR**

### CODE REVIEW

File: astro/create-and-link-connections.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ description: "Create Airflow connections and link them to multiple Deployments i
 
 You can create and manage Airflow connections for Deployments with the Astro Environment Manager in the Cloud UI. The Environment Manager uses an Astro-managed secrets backend to store connection configurations as Kubernetes Secrets.
 
+:::info Note
+
+Astro Environment Manager is only available to Astro Hosted users; it is not available for Astro Hybrid users.
+
+
```
Comment (jwitz) [inline astro/create-and-link-connections.md:14]:
```suggestion
:::info

The Astro Environment Manager is available only on Astro Hosted

:::
```

Comment (jwitz) [reply astro/create-and-link-connections.md:14]: This is smaller, but it might be more visually distinctive and therefore visible for readers

**Review by jwitz — APPROVED** [review]
LGTM @RNHTTR , hope you don't mind but I'd like to try using our Hosted badge HTML element here instead of a full note!

File: astro/create-and-link-connections.md (hunk 1 above)
Comment (RNHTTR) [reply astro/create-and-link-connections.md]: I'm not opposed to the hosted badge, but should it say something more obvious like "Hosted Only"? I'm not sure a Hybrid user would find the badge to obviously imply it's hosted-only.

Comment (jwitz) [reply astro/create-and-link-connections.md]: Good point, we can work on creating a separate "Hosted only" badge in the next quarter to apply on docs.

**Pull request merged**